test/models/setting_test.rb

```diff
@@ -561,4 +560,41 @@ def check_value_returns_from_cache_with(options = {})
     bmc.value = false
     refute_valid bmc, :base, 'Unable to disable safemode_render when bmc_credentials_accessible is disabled'
   end
+
+  context "when email.yaml exists" do
```

Comment:
These should be in a test case specifically for Setting::Email, probably in `test/models/setting/`?